cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/messages/GroupedMessageBatchInserter.java

```diff
@@ -55,18 +56,14 @@ public CompletableFuture<AsyncResultSet> insert(GroupedMessageBatchEntity groupe
                 .setLocalTime(FIELD_LAST_MESSAGE_TIME, groupedMessageBatch.getLastMessageTime())
                 .setInt(FIELD_MESSAGE_COUNT, groupedMessageBatch.getMessageCount())
                 .setBoolean(FIELD_COMPRESSED, groupedMessageBatch.isCompressed())
+                .setSet(FIELD_LABELS, groupedMessageBatch.getLabels(), String.class)
                 .setByteBuffer(FIELD_CONTENT, groupedMessageBatch.getContent())
                 .setInstant(FIELD_REC_DATE, Instant.now())
                 .setInt(FIELD_CONTENT_SIZE, groupedMessageBatch.getContentSize())
                 .setInt(FIELD_UNCOMPRESSED_CONTENT_SIZE, groupedMessageBatch.getUncompressedContentSize());
 
-        // We skip setting null value to columns to avoid tombstone creation.
-        if (groupedMessageBatch.getLabels() != null) {
-            builder = builder.setSet(FIELD_LABELS, groupedMessageBatch.getLabels(), String.class);
-        }
-
-        attributes.apply(builder);
-        BoundStatement statement = builder.build();
+        attributes.apply(builderWrapper.getUnderlyingBuilder());
```

Comment:
Why we use getUnderlyingBuilder instead of our builderWrapper?

Reply:
to reduce the number of changes required